python/tests/unit/test_import_hygiene.py

```
@@ -0,0 +1,65 @@
+        "assert pkg.Monitor is not None; "
+        "assert pkg.Config is not None; "
+        "assert pkg.load_config is not None"
+    )
+    assert result.returncode == 0, result.stderr or result.stdout
+
+
+def test_minimal_install_imports_package(tmp_path):
+    venv = tmp_path / "venv"
+    subprocess.run([sys.executable, "-m", "venv", str(venv)], check=True)
+    pip = venv / "bin" / "pip"
+    subprocess.run([str(pip), "install", "-e", str(_PYTHON_ROOT)], check=True)
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)